docs/sources/enterprise/auditing.md

```diff
@@ -0,0 +1,166 @@
+
+- **timestamp\* -** _string -_ The date and time the request was made, in coordinated universal time (UTC) using the [RFC3339](https://tools.ietf.org/html/rfc3339#section-5.6) format.
+- **user\* -** _object -_ Information about the user that made the request. At least one of the `UserID` / `ApiKeyID` fields will not be empty if `isAnonymous=false`.
+  - **userId -** _number -_ ID of the Grafana user that made the request.
+  - **orgId\* -** _number -_ Current organization of the user that made the request.
+  - **orgRole -** _string -_ Current role of the user that made the request.
+  - **name -** _string -_ Name of the Grafana user that made the request.
+  - **apiKeyId -** _number -_ ID of the Grafana API key used to make the request.
+  - **isAnonymous\* -** _boolean -_ `true` if an anonymous user made the request, `false` otherwise.
+- **action\* -** _string -_ The request action (eg. `create`, `update`, `manage-permissions`).
+- **request\* -** _object -_ Information about the HTTP request.
+  - **params -** _object -_ Request path parameters.
+  - **query -** _object -_ Request query parameters.
+  - **body -** _string -_ Request body.
+- **Result\* -** _object -_ Information about the HTTP response.
+  - **statusType\* -** _string -_ `success` if the request action was successful, `failure` otherwise.
+  - **statusCode -** _number -_ HTTP status of the request.
+  - **failureMessage -** _string -_ HTTP error message.
+  - **body -** _string -_ Response body.
+- **resources -** _array -_ Information about the resources that the request action impacted. Can be null for non-resource actions like `login` and `logout`.
+  - **id\* -** _number -_ ID of the resource.
+  - **type\* -** _string -_ Type of the resource (logged resources are: `alert`, `alert-notification`, `annotation`, `api-key`, `auth-token`, `dashboard`, `datasource`, `folder`, `org`, `panel`, `playlist`, `report`, `team`, `user`, `version`).
+- **requestUri\* -** _string -_ Request URI.
+- **ipAddress\* -** _string -_ IP address that the request was made from.
+- **userAgent\* -** _string -_ Agent through which the request was made.
+- **grafanaVersion\* -** _string -_ Grafana current version when this log is created.
+- **additionalData -** _object -_ Provide additional information on the request. For now, it's only used in `login` actions to log external user information if an external system was used to log in.
+
+### Recorded actions
+
+The audit logs include records about the following categories of actions:
+
+**Sessions**
+
+- Login
+- Logout
+- Revoke a user authentication token
+- Create or delete an API key
+
+**User management**
+
+- Create, update or delete a user
+- Enable or disable a user
+- Manage user role and permissions
+- LDAP sync or information access
+
+**Team and organization management**
+
+- Create, update or delete a team / organization
+- Add or remove a member to a team / organization
+- Manage team / organization members roles
```

Comment:
```suggestion
- Manage team or organization member roles
```

Reply:
I'm not sure this makes sense. Team members do not have roles. Only the organization does.

Reply:
Team members have roles if the `editors_can_admin` [feature](https://grafana.com/docs/grafana/latest/permissions/organization_roles/#editor-role) is enabled (they can manage the team).

Reply:
They are still editors, though. And managing the team != specific team member roles, unless I am totally missing something.

Reply:
As a user that is an editor but have the role team admin on a team I can make other team members into team admins.

Reply:
What I mean by that is making a team Member an Admin of a team (or a Member if he was Admin before). This is an action logged by the auditing feature. Maybe "Manage team or organization member permissions" would be more accurate? Or should I split it into 2 sentences to talk about "organization role" and "team permission"?

Reply:
I think splitting that bullet into two would help.